sequelize/models/resolver.js

```diff
@@ -2,8 +2,8 @@ module.exports = (sequelize, DataTypes) => {
   const Resolver = sequelize.define('Resolver', {
     type: DataTypes.STRING,
     field: DataTypes.STRING,
-    requestMapping: DataTypes.STRING,
-    responseMapping: DataTypes.STRING
+    requestMapping: DataTypes.TEXT,
+    responseMapping: DataTypes.TEXT
```

Comment:
These 2 were causing problems. STRING by default has a capacity of 255 chars in POSTGRES.
Using TEXT means unlimited capacity.